ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -64,6 +64,9 @@
 flowtests = pytest.mark.flowtests
 system_test = pytest.mark.system_test
 performance = pytest.mark.performance
+performance_a = pytest.mark.performance_a
+performance_b = pytest.mark.performance_b
+performance_c = pytest.mark.performance_c
```

Comment:
@Avilir is it based on test run_time?
We need to inform the Eco-System team on this since it needs updates in the regression runs tier markers.

Reply:
Run time of each marker is about 1/3 of the full suite run.